include/deal.II/multigrid/multigrid.h

```diff
@@ -99,9 +99,8 @@ class Multigrid : public Subscriptor
             Cycle                              cycle = v_cycle);
 
   /**
-   * Experimental constructor for cases in which no DoFHandler is available.
-   *
-   * @warning Not intended for general use.
+   * Constructor. Same as above but you can determine the multigrid
+   * levels to use yourself.
```

Comment:
The `Multigrid` class doesn't use a `DoFHandler` apart from setting default values for the levels we want to work on. Therefore, I like having a constructor that can be used without providing a `DoFHandler`. Unfortunately, none of the other objects that are used in the constructor provides us with the minimum and maximum level they have. I would be in favor of changing this, i.e. get this information from ` MGMatrixBase`.

I don't care too much what we do in the other case. If you are in favor of having the possibility to specify `minlevel` and `maxlevel` in the first constructor as well, I am fine with including it.